build.gradle

```diff
@@ -104,11 +104,19 @@ task exportSampleSphereBlast {
     }
 }
 
+
 task buildTestApp {
     description 'Creates the instant app test project, then builds a test app'
     def buildFunction = 'GooglePlayInstant.Samples.TestApp.Editor.TestAppBuilder.Build'
     def buildPath = "${playInstantBuildPath}/testapp"
     def logFile = file("testapp.log").getPath()
+    
+    //Specify output file path relative to the root path using -PoutputFile
+    def outputFileArg = project.hasProperty('outputFile') ? outputFile : "${playInstantBuildPath}/testApp.apk"
+    outputFileArg = file(outputFileArg)
+    
+    println 'Building to: '+outputFileArg
```

Comment:
Prefer "Building to ${outputFileArg}"

Reply:
Done

Reply:
I also moved that line into the DoLast{} block because it was running when I was calling other tasks.